### PR TITLE
Added ssl_ca_path and ssl_cert_store to client global options

### PR DESCRIPTION
**Pull request by Bajena**
Some people might find it useful to use ca_cert path and cert store, like in Faraday

### CODE REVIEW

**Comment by Bajena** [comment]
Checkout https://github.com/savonrb/httpi/pull/165

File: lib/savon/request.rb [hunk 1]
```diff
@@ -94,4 +96,4 @@ def configure_headers(soap_action)
       @http_request.headers["Content-Type"] ||= CONTENT_TYPE[@globals[:soap_version]] % @globals[:encoding]
     end
   end
-end
+end
```
Comment (olleolleolle) [inline lib/savon/request.rb]: Don't remove the file-ending newline.

File: spec/savon/options_spec.rb [hunk 2]
```diff
@@ -1051,5 +1071,4 @@ def inspect_request(response)
     hash = JSON.parse(response.http.body)
     OpenStruct.new(hash)
   end
-
-end
+end
```
Comment (olleolleolle) [inline spec/savon/options_spec.rb]: Don't remove the file-ending newline. (Here, too.)

**Comment by stupied4ever** [comment]
Any update on that? It is really important for me :(

**Comment by Bajena** [comment]
Closing in favor of https://github.com/savonrb/savon/pull/838

**Pull request closed without merging**